app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java

```diff
@@ -238,6 +241,15 @@ protected void addLatestMovies() {
 
     }
 
+    protected void addLatestTVShows() {
+        LatestItemsQuery latestTVShows = new LatestItemsQuery();
+        latestTVShows.setFields(new ItemFields[] {ItemFields.PrimaryImageAspectRatio, ItemFields.Overview});
+        latestTVShows.setIncludeItemTypes(new String[]{"Series"});
```

Comment:
To get results that are more consistent with the web interface, it looks like we should include `Episode` instead of `Series` and set `latestTVShows.setGroupItems(true);`.

Reply:
Is there a way to force widescreen tiles? 

Doing it this way leads to a bug where if there is only one episode left in a season, the widescreen card is shown, for the grouped ones the portrait.